Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:14-slim
+
+WORKDIR /usr/src/app
+
+COPY Backend/package*.json ./
+
+RUN npm install
+
+COPY Backend/. /usr/src/app/Backend/
```

Comment:
Copy whole application instead of just the backend

Reply:
Handled in frontend docker file